app/helpers/guide_helper.rb

```diff
@@ -16,16 +16,14 @@ def state_label(guide)
 
   def guide_action_button(guide)
     state = guide.latest_edition.state
-    if state == "draft" || state == "published"
-      title = "Edit"
-      path =  edit_guide_path(guide)
-    elsif state == "review_requested"
-      title = "Review guide"
-      path = edition_path(guide.latest_edition)
-    elsif state == "approved"
-      title = "Publish"
-      path = edition_path(guide.latest_edition)
-    end
+    title = if state == "draft" || state == "published"
```

Comment:
I think this may look a lot nicer as a case statement or even a hash
```ruby
title = { 
  "draft" => "Edit", 
  "review_requested" => "Review guide" ... 
}[state]
```